src/pluggy/callers.py

```diff
@@ -200,9 +208,17 @@ def _multicall(hook_impls, caller_kwargs, firstresult=False):
         # run all wrapper post-yield blocks
         for gen in reversed(teardowns):
             try:
-                gen.send(outcome)
-                _raise_wrapfail(gen, "has second yield")
-            except StopIteration:
-                pass
+                try:
+                    gen.send(outcome)
```

Comment:
`gen.send()`/`gen.throw()` alternative here should generate more informative tracebacks. By the cost of adding `finally:` to hook wrappers  that do not expect exception...